src/pages/QueryLogs/index.tsx

```diff
@@ -34,11 +85,76 @@ const QueryLogs: React.FC<QueryLogsProps> = ({ logs }) => {
           }}
         />
         <div className={styles.body}>
-          <QueryLogsTable logs={logs} />
+          <BouncingDotsLoader loading={fetching}>
+            <Space size={27} className={styles.space} direction="vertical">
+              <QueryFilters
+                defaultValues={defaultFilterState}
+                values={filter}
+                dataSources={dataSources}
+                onChange={onFilterUpdate}
+              />
+              <QueryLogsTable
+                logs={logs}
+                onClickRow={onClickRow}
+                pagination={{
+                  pageSize,
+                  current: currentPage,
+                  total: logsCount,
+                  onChange: (current) => onPageChange({ current }),
+                }}
+              />
+            </Space>
+          </BouncingDotsLoader>
         </div>
       </Space>
-    </AppLayout>
+    </SidebarLayout>
+  );
+};
+
+const QueryLogsWrapper = () => {
+  const [filter, setFilter] = useState(defaultFilterState);
+
+  const { withAuthPrefix } = useAppSettings();
+  const [, setLocation] = useLocation();
+  const basePath = withAuthPrefix("/logs/requests");
```

Comment:
why url is `/logs/requests/` but page url is `/logs/query`? Base path should be the same